core/iwasm/fast-jit/fe/jit_emit_function.c

```diff
@@ -247,6 +350,10 @@ jit_compile_op_call(JitCompContext *cc, uint32 func_idx, bool tail_call)
 
     /* Ignore tail call currently */
     (void)tail_call;
+
+    /*free argvs */
+    jit_free(argvs);
+
     return true;
 fail:
```

Comment:
Should also free argvs if it is not NULL.

Reply:
OK,I will check my code.